packages/core/test/lib/commands/debug.js

```diff
@@ -25,25 +23,31 @@ describe("debug", function () {
     });
   });
 
-  describe("mergeConfigNetwork(config, options)", function () {
+  describe.only("mergeConfigNetwork(config, options)", function () {
```

Comment:
get rid of this `only`

Reply:
Oh! I forgot that. I will remove it right now!